test/test_ssltransport.py

```diff
@@ -503,3 +503,65 @@ def test_constructor_params(self):
             mock.ANY, mock.ANY, server_hostname=server_hostname
         )
         assert not ssl_transport.suppress_ragged_eofs
+
+    def test_recv_no_flags_error(self):
```

Comment:
Since all of these are basically mocking out and using the same setup, how about combining them into one test that asserts all the things with `flags=...`?